examples/simple-app/yarn.lock

```diff
@@ -1,15218 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Should we not keep this file?

Reply:
No, since `simple-app` is registered as a workspace it should not have its own `yarn.lock` file in the first place: https://github.com/dhis2/app-platform/blob/master/package.json#L11

Reply:
Ah ok. Well seems good to merge then 👍